py/selenium/webdriver/common/service.py

```diff
@@ -93,7 +93,7 @@ def start(self) -> None:
                 logger.debug("driver not found in PATH, trying Selenium Manager")
                 browser = self.__class__.__module__.split(".")[-2]
                 try:
-                    path = SeleniumManager.driver_location(browser)
+                    path = SeleniumManager().driver_location(browser)
                 except WebDriverException as new_err:
                     logger.debug("Unable to obtain driver using Selenium Manager: " + new_err.msg)
```

Comment:
I _think_ this is sufficient to propagate here from the changes, should include the message tho unit tests do not currently test an actual `service` based flow; only exercising the manager directly at the moment.

Reply:
yeah, we're going to move to instantiating this class for next set of features, I think.